packages/web-api-scan-runner/src/crawler/discovered-urls-processor.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { injectable } from 'inversify';
+import _, { isNil } from 'lodash';
+import { WebsiteScanResult } from 'storage-documents';
+
+@injectable()
+export class DiscoveredUrlsProcessor {
```

Comment:
It seems like this doesn't need to be a class since it maintains no state data. We could just export the main function.